themes/gatsby-theme-blog-core/gatsby-node.js

```diff
@@ -53,11 +42,24 @@ const mdxResolverPassthrough = fieldName => async (
   })
   return result
 }
-exports.sourceNodes = ({ actions, schema }) => {
+
+exports.createSchemaCustomization = ({ actions, schema }) => {
   const { createTypes } = actions
+  createTypes(`interface BlogPost @nodeInterface {
+      id: ID!
+      title: String!
+      body: String!
+      slug: String!
+      date: Date! @dateformat(formatString: "DD-MM-YYYY")
```

Comment:
```suggestion
      date: Date! @dateformat
```
Or, if we want a default format string, we should have it on `MdxBlogPost` as well